### ISSUE TITLE
🐛 Expansive Embed not Splitting Pages Correctly

### ISSUE THREAD

**Issue by YousefEZ** [YousefEZ/discord-qalib#161]
**Describe the bug**
Expansive Embed does not take into account the title, or other *static* attributes of the menu, so it will always cross the threshold.

**To Reproduce**
1. Submit template file causing the behavior 
2. Submit the code that you are using to run it

```xml
<discord>
    <expansive key="test">
        <title>Test Title</title>
        <colour>magenta</colour>
        <expansive_field>
            <name>Page</name>
            <value>{% for c in range(1024) %}c{% endfor %}</value>
        </expansive_field>
    </expansive>
</discord>
```

**Expected behavior**
For the character count to be subtracted by static elements

**Screenshots**

**Server Platform (if relevant)**
- [x] Windows
- [x] MacOS
- [x] Linux

**Viewing Platform**
- [x] Windows
- [x] MacOS
- [x] Linux
- [x] iOS
- [x] Android

**Additional context**

**Comment by YousefEZ** [comment]
To remedy this issue, we will change it become `Expansive Embed` rather than just the field, so that the entire embed can get access to the Page number key, as well

**Comment by YousefEZ** [comment]
Fixed in #162
